### ISSUE TITLE
implement_methods false positive: does not consider ABSTRACT METHODS syntax

### ISSUE THREAD

**Issue by FreHu** [abaplint/abaplint#1137]
![](https://user-images.githubusercontent.com/5097067/89293526-5a641600-d65e-11ea-8551-e09486b5a6c8.png)

**Comment by larshp** [comment]
reproduced as

```abap
INTERFACE lcl_intf.
  METHODS method_name.
ENDINTERFACE.

CLASS lcl_bar DEFINITION ABSTRACT.
  PUBLIC SECTION.
    INTERFACES lcl_intf ABSTRACT METHODS method_name.
ENDCLASS.

CLASS lcl_bar IMPLEMENTATION.
ENDCLASS.
```

![image](https://user-images.githubusercontent.com/5888506/89704841-ea080e00-d957-11ea-80c2-046c328f9b9d.png)

**Comment by larshp** [comment]
try again, 2.38.17
